profiler/options.go

```diff
@@ -71,6 +74,19 @@ func urlForSite(site string) (string, error) {
 	return u, err
 }
 
+// Check for API key constraints:
+func apiKeyCheck(key string) bool {
```

Comment:
Let's call this something else, like `isAPIKeyValid`. Also, in Go, documentation needs to start with the name of the identifier that it documents, so:
```go
// isAPIKeyValid reports whether the given key is a valid API key.
```

Reply:
Same here: It would be great if you could only resolve a comment thread after it was addressed, or allow the reviewer to resolve it.

Reply:
Agreed and addressed as suggested in ef1b11d